DQM/SiPixelMonitorClient/src/SiPixelActionExecutor.cc

```diff
@@ -202,7 +203,9 @@ void SiPixelActionExecutor::createSummary(DQMStore::IBooker & iBooker, DQMStore:
 //=============================================================================================================
 void SiPixelActionExecutor::bookDeviations(DQMStore::IBooker & iBooker, bool isUpgrade) {
   int nBPixModules;
+  int nFPixModules;
   if (isUpgrade) {nBPixModules=1184;} else {nBPixModules=768;} 
+  if (isUpgrade) {nFPixModules=672;} else {nFPixModules=672;} 
```

Comment:
It doesn't. It's just `nFPixModules=672`. Please don't use plain numbers like that (especially if they are used multiple times across code base). Add const static or pre-processor macro defining it and adding a comment line explaining what it means.